plugin-ci/orb.yml

```diff
@@ -66,12 +65,17 @@ aliases:
 
 jobs:
   lint:
+    parameters:
+      version:
+        default: 1.24.0
```

Comment:
removing the default from above (making it a required parameter) and putting the default here -- this keeps the default value in one place for future changes.